pyfluidsynth.py

```diff
@@ -36,9 +39,9 @@
 
 lib = find_library('fluidsynth') or \
     find_library('libfluidsynth') or \
-    find_library('libfluidsynth-3') or \
```

Comment:
Please revert this change and rebase.  It is best to install the more current library `libfluidsynth-3` before trying earlier libraries (`libfluidsynth-2` and `libfluidsynth-1`).